kefir.js.flow

```diff
@@ -0,0 +1,160 @@
+/* @flow */
+
+export type Event<T> =
+  {type: 'value', value: T} |
+  {type: 'error', value: any} |
```

Comment:
I think error type should also be parameterized. Did you see https://github.com/flowtype/flow-typed/blob/master/definitions/npm/kefir_v3.x.x/flow_%3E%3Dv0.20.x/kefir_v3.x.x.js btw?